src/pineko/evolve.py

```diff
@@ -5,32 +5,50 @@
 
 import eko
 import eko.basis_rotation as br
-import eko.compatibility
 import numpy as np
 import pineappl
 import rich
 import rich.box
 import rich.panel
 import yaml
+from eko.io.types import EvolutionMethod
 
 from . import check, comparator, ekompatibility, version
 
-DEFAULT_PIDS = [-5, -4, -3, -2, -1, 21, 22, 1, 2, 3, 4, 5]
+
+def sv_scheme(tcard):
+    """Infere the factorization scale_variation scheme to be used from the theory card.
+
+    Parameters
+    ----------
+    tcard : dict
+        theory card
+
+    """
+    modsv_list = ["expanded", "exponentiated"]
```

Comment:
https://github.com/NNPDF/eko/blob/873dc2d3cf68c12058db80f067e5d4ef700fcf2c/src/eko/io/types.py#L160-L164

Do not repeat existing values, this is:
```suggestion
    modsv_list = {a.value for a in ScaleVariationsMethod}
```
(and I put braces since you need a set, not a list)